kogito-build/kogito-build-no-bom-parent/pom.xml

```
@@ -671,6 +670,23 @@
             <legacyMode>true</legacyMode>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.commonjava.maven.plugins</groupId>
+          <artifactId>directory-maven-plugin</artifactId>
+          <version>0.1</version>
```

why not to use latest one?
```suggestion
          <version>1.0</version>
```